_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -0,0 +1,311 @@
+    path and return its xpub along with a signature generated by its
+    corresponding private key.  The set of {identifying_parameters,
+    key_path, xpub, signature} are encrypted and returned to the
+    coordinator by each wallet.  The coordinator then combines them into
+    an [output script descriptor][topic descriptors] that is encrypted
+    and passed back to each member wallet, who verifies their xpub is
+    included and then stores the descriptor as a script template they're
+    willing to sign for.
+
+    The proposal received a significant amount of discussion on list,
+    and some changes are planned.  We'll monitor the discussion for
+    significant updates as the proposal matures and gets closer to
+    implementation.
+
+- **Escrow over LN:** a discussion [started][zmn escrow1] on the
+  Lightning-Dev mailing over a year ago about creating non-custodial
+  onchain escrows for LN saw [renewed discussion][aragoneses escrow]
+  this past week.  A particular standout in the discussion was a
+  [post][zmn escrow2] by ZmnSCPxj that uses one of [De Morgan's laws][]
+  for boolean statements to greatly simply the construction of
+  escrows---and [possibly][kohen escrow] many other LN-based
+  contracts---with the tradeoff of requiring a seller to post a bond
+  before receiving payment.  The operation requires an upgrade of LN to
+  use [PTLCs][topic ptlc], which is not currently expected until after
+  [taproot][topic taproot] activates.
+
+- **Renewed discussion about bidirectional upfront LN fees:**  Joost
+  Jager [restarted][jager hold fees] Lightning-Dev mailing list
+  discussion about adding LN service fees that charge spenders and
+  receivers for the time they consume using a channel's limited
+  availability of funding ("liquidity") and concurrent payment capacity
+  ("HTLC slots").  Jager builds on a previous proposal (see [Newsletter
+  #122][news122 bidir]), extending its fixed fees with fees
+  proportional to payment processing duration ("hold fees").  The
+  proposal received moderate discussion, with one [concern][zmn hold fee
+  privacy] being a reduction in sender/receiver privacy.  The underlying
+  issue here has been [known][russell loop] for five years and discussed
+  at length many times, so we expect discussion to continue.
+
+- **Anti-exfiltration:** Andrew Poelstra and Jonas Nick published a blog
+  [post][anti-exfil] about a security technique being implemented for
+  the Shift [BitBox02][] and Blockstream [Jade][] hardware wallets.  The
+  goal is to allow both a hardware wallet and the computer controlling
+  it to each be assured that the nonce used to generate a signature is
+  actually an unguessable value.  This prevents malicious hardware
+  wallet firmware from generating nonces known to the firmware author,
+  which the author could combine with your signatures found onchain to
+  derive your private keys, allowing them to spend any other bitcoins
```

Comment:
the shift from 3rd to 2nd person in this sentence (using "your" twice instead of user/client/node/etc.) is a bit surprising; it's not used elsewhere in this newsletter